config/functions.cfg

```diff
@@ -222,7 +223,8 @@ function git_clone {
   echo -e
   echo -e "${GREEN}Cloning into ${CYAN}mx-chain-go${GREEN} with tag ${CYAN}$BINARYVER${GREEN}...${NC}"
   echo -e
-  cd $GOPATH/src/github.com/multiversx && git clone https://github.com/multiversx/mx-chain-go && cd mx-chain-go && git checkout --force $BINARYVER
+  BINARYVER_SHORT=$(echo $BINARYVER | sed 's/tags\///')
+  cd $GOPATH/src/github.com/multiversx && git clone https://github.com/multiversx/mx-chain-go --branch=$BINARYVER_SHORT --single-branch --depth=1
```

Comment:
this does not work with `tags/v1.4.4`
Also, how could we use the `force` option?

Reply:
LE: works, because of the seds on L216/L226/L240